index.html

```diff
@@ -57,11 +57,227 @@ <h3 class="connect">LET'S CONNECT</h3>
             </li>
 
             <li>
-              <a href="#"><img src="./images/Vector.svg" alt="Two-fingers"></i></a>
+              <a href="#"
+                ><img src="./images/Vector.svg" alt="Two-fingers"
+              /></a>
             </li>
           </ul>
         </div>
       </section>
     </main>
+
+    <!-- WORKS -->
+    <section class="works">
+      <div class="contents container">
+        <div class="projects">
+          <div class="details">
+            <img src="./images/Snapshoot-Portfolio.png" alt="" />
+
+            <div class="card-info">
+              <h2>TONIC</h2>
+
+              <div class="title-and-date">
+                <p class="title">CANOPY</p>
+                <span></span>
+                <p class="muted">Back End Dev</p>
+                <span></span>
+                <p class="muted">2015</p>
+              </div>
+
+              <p class="description">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+
+              <ul class="stack">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>JAVASCRIPT</li>
+              </ul>
+
+              <a class="btn" href="#">See projects</a>
+            </div>
+          </div>
+
+          <div class="details">
+            <img src="./images/Snapshoot-Portfolio1.png" alt="" />
```

Comment:
- You have done well making your site responsive, but to further improve the UI, I suggest making the project images responsive with respect to the project cards. Please fix this 🙏 
Your project:
![rwgwg](https://user-images.githubusercontent.com/73607512/227505064-61837a08-1a12-4dd6-bb88-0efd3288876c.PNG)

Reply:
How on earth did I forget this? 🤨
Fixed. Thanks so much.